api/RS.API/RS.Data/Logic/CandidateRepository.cs

```diff
@@ -87,5 +87,10 @@ public void ApprovedForInterview(Candidates candidate)
             _context.Entry(candidate).State = EntityState.Modified;
             _context.SaveChanges();
         }
+
+        public List<string> GetOrganizationsOnInputChanged(string input)
+        {
+            return _context.Organizations.Where(x => x.Name.Substring(0, input.Length).Equals(input) && (x.IsActive && !x.IsDeleted)).Select(x => x.Name).ToList();
```

Comment:
why substring is used here ?  Only string list is sent ? How you are managing the id of Organization ?